docroot/themes/custom/twentyeighteen/sass/base/_base.scss

```diff
@@ -7,16 +7,18 @@ body {
 }
 
 body {
-  position: relative;
-  font-family: $body_font;
-  font-size: 1em;
   color: $dark-gray;
+  font-size: 1em;
+  font-family: 'Montserrat', Helvetica, Arial, sans-serif;
   text-rendering: optimizeLegibility;
   font-weight: 400;
   line-height: 1.4;
   margin: 0;
   padding: 0;
-  background-color: rgba($teal, 0.5);
+  background-color: rgba(54, 114, 122, 0.5);
```

Comment:
Let's keep it at `rgba($teal, 0.5)` because this takes the RBG value of $teal (in _config.scss) and transforms it to and RGBA with a transparency.